goto_commands.py

```diff
@@ -0,0 +1,136 @@
+import sublime
+import sublime_plugin
+
+from itertools import dropwhile, takewhile
+from .lint import persist
+
+
+"""
+Implement typical Goto Next Previous Error Commands. If you will, add
+the following to your key bindings.
+
+```
+  { "keys": ["ctrl+k", "n"],
+    "command": "sublime_linter_goto_error",
+    "args": {"direction": "next"}
+  },
+  { "keys": ["ctrl+k", "p"],
+    "command": "sublime_linter_goto_error",
+    "args": {"direction": "previous"}
+  },
+```
+
+Supported args:
+
+* `direction`: 'previous' | 'next'
+* `count`: How wide you will jump. Defaults to 1.
```

Comment:
What's the idea behind this? Is that so, perhaps from the command palette, you can say "jump 5 errors down"? Or is it just for the vim-style chords for neovintageous?

Reply:
Yes, you could do that. 

I merely implemented that bc it was specced, and I immediately thought that it is worth the extra effort.